pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -188,6 +189,7 @@ def on_save_checkpoint(self, trainer, pl_module) -> Dict[str, Any]:
             "monitor": self.monitor,
             "best_model_score": self.best_model_score,
             "best_model_path": self.best_model_path,
+            "current_score": self.current_score,
```

Comment:
```suggestion
            "checkpoint_score": self.current_score,
```
maybe `checkpoint_score`?

Reply:
IMO thats redundant. Its in a class with checkpoint in its name.

In any case, we could do `current_model_score` to be consistent with `best_model_score`. But I find `current_score` simpler.

Reply:
cool